nck/readers/gcs_reader.py

```diff
@@ -39,14 +39,10 @@ def gcs(**kwargs):
 
 class GCSReader(ObjectStorageReader, GoogleBaseClass):
     def __init__(self, bucket, prefix, format, dest_key_split=-1, **kwargs):
-        super().__init__(
-            bucket, prefix, format, dest_key_split, platform="GCS", **kwargs
-        )
+        super().__init__(bucket, prefix, format, dest_key_split, platform="GCS", **kwargs)
 
     def create_client(self, config):
-        return storage.Client(
-            credentials=self._get_credentials(), project=config.project_id
-        )
+        return storage.Client(credentials=self._get_credentials(), project=config.project_id)
```

Comment:
I definitely don't like this way to call the GCS project_id. Could we find another to call it?

Reply:
There's already an issue on this topic: #93 .

Reply:
Yes, I forgot to resolve this conversation, thanks!